app/Platform/Jobs/UpdatePlayerPlayedGamesJob.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Platform\Jobs;
+
+use App\Models\User;
+use App\Platform\Actions\UpdatePlayerPlayedGames;
+use Illuminate\Bus\Batchable;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldBeUniqueUntilProcessing;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+
+class UpdatePlayerPlayedGamesJob implements ShouldQueue, ShouldBeUniqueUntilProcessing
+{
+    use Batchable;
+    use Dispatchable;
+    use InteractsWithQueue;
+    use Queueable;
+    use SerializesModels;
+
+    public function __construct(
+        private readonly int $userId,
+    ) {
+    }
+
+    public int $uniqueFor = 3600;
+
+    public function uniqueId(): string
+    {
+        return config('queue.default') === 'sync' ? '' : $this->userId;
+    }
+
+    /**
+     * @return array<int, string>
+     */
+    public function tags(): array
+    {
+        return [
+            User::class . ':' . $this->userId,
+        ];
+    }
+
+    public function handle(): void
+    {
+        if ($this->batch()?->cancelled()) {
+            return;
+        }
+
+        $user = User::find($this->userId);
+
+        if (!$user) {
+            // might've been deleted
+            return;
+        }
+
+        app()->make(UpdatePlayerPlayedGames::class)
+            ->execute(User::findOrFail($this->userId));
```

Comment:
nit: I can't help but notice we're fetching the user twice here. Not sure why I didn't notice before. Seems like this is in happening both in this class and in _UpdatePlayerMetricsJob.php_.

Reply:
Copied from `UpdatePlayerMetricsJob`. Updated both.